dashboard/calibration_manager.py

```diff
@@ -38,8 +58,28 @@ def convert_exp_to_sim(self, exp_dict):
             )
             # fill the dictionary
             if exp_name in exp_dict:
-                sim_val = (exp_dict[exp_name] - value["beta"]) * value["alpha"]
-                sim_dict[sim_name] = sim_val
+                if state.use_inferred_calibration:
+                    sim_dict[sim_name] = (
+                        exp_dict[exp_name] - value["beta_guess"]
+                    ) * value["alpha_guess"]
```

Comment:
What is `sim_dict[sim_name]` set to if `state.use_inferred_calibration` returns `False`?